docs/storage/gridftp.md

```diff
@@ -1,168 +1,123 @@
-
-If you have a multi-homed host you may be interested in reading [this section](#ConfigMultiHomed).
-
-—\#\# Engineering Considerations
-
-It is recommended that the GridFTP package be installed on its own server if:
-
--   You are serving the VOs that use storage heavily (CMS, ATLAS, CDF, and D0) and have more than 250 cores
+-   You are serving VOs that use storage heavily (CMS, ATLAS, CDF, and D0) and your site has more than 250 cores
 -   Your site will be managing more than 50 TB of disk space
+-   A simple front-end to a filesystem allowing access over WAN - for example NFS.
 
-If you are planning to have a Storage Element with BeStMan and have more than 1Gbps bandwidth, then you should plan on at least one GridFTP server per 4Gbps of available bandwidth (assuming you have 10Gbps interfaces on the server) if you want to maximize throughput.
-
-Also, you have to decide what authorization mechanism you prefer. You may use either grid-mapfile or a GUMS server for users’ authentication and authorization. We currently recommend using GUMS as it provides superior flexibility and allows a site to manage all of its mappings in one central location; most large sites use GUMS.
-
-<span class="twiki-macro NOTE"></span> OSG does not support launching the GridFTP server with xinetd — only launching with init is supported.
+!!! note
+    This document is for a standalone GridFTP server on top of POSIX storage.  [See this page](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/InstallHadoop200SE) for installation and configuration of a GridFTP server on top of the Hadoop Distributed File System.
 
-—\# Install Instructions
+Before Starting
+---------------
 
-<span class="twiki-macro INCLUDE" section="OSGRepoBrief" TOC_SHIFT="+">YumRepositories</span> <span class="twiki-macro INCLUDE" section="OSGBriefCaCerts" TOC_SHIFT="+">InstallCertAuth</span>
+Before starting the installation process you will need to fulfill these prerequisites.
 
-<span class="twiki-macro STARTSECTION">Full</span>
 
-<span class="twiki-macro STARTSECTION">Install</span> GridFTP requires a certificate package to run. If you require a specific certificate package, follow the Documentation/Release3.InstallCertAuth instructions to install it. If you do not install a grid certificate package first, the install procedure will install one for you as part of its dependencies. (usually osg-ca-certs).
+-   Ensure the host has [a supported operating system](../release/supported_platforms.md)
+-   Obtain root access to the host
+-   Prepare [the required Yum repositories](../common/yum.md)
+-   Install [CA certificates](../common/ca.md)
+-   Service certificate: The HTCondor-CE service uses a host certificate at `/etc/grid-security/hostcert.pem` and an accompanying key at `/etc/grid-security/hostkey.pem`
```

Comment:
Yeah - there were a lot of copy/pastes from the HTCondor-CE document in the original source.  Must have missed this one.